### PR TITLE
Reference to auth source in mongo url

### PR DESCRIPTION
**Pull request by zub0r**
Fixes https://github.com/prisma/prisma/discussions/9994

### CODE REVIEW

**Comment by janpio** [comment]
Thanks for the suggestion.

Can you help us understand what exactly that means? 

Per documentation from MongoDB https://www.mongodb.com/docs/manual/reference/connection-string/#mongodb-urioption-urioption.authSource `authSource=admin` should not actually be necessary as it is one of the fallbacks, right?

**Comment by zub0r** [comment]
The connection didn't work for me without specifying it explicitly. I was using mongo 5.0.

File: content/100-getting-started/02-setup-prisma/200-add-to-existing-project/120-mongodb/100-connect-your-database.mdx [hunk 1]
```diff
@@ -46,6 +46,8 @@ As an example, for a MongoDB database hosted on [MongoDB Atlas](https://www.mong
 DATABASE_URL="mongodb+srv://test:test@cluster0.ns1yp.mongodb.net/myFirstDatabase"
 ```
 
+Optionally, you may specify the source database for the authentication by [adding](https://github.com/prisma/prisma/discussions/9994#discussioncomment-1562283) `?authSource=admin` to the URL.
```
Comment (tanberry) [inline content/100-getting-started/02-setup-prisma/200-add-to-existing-project/120-mongodb/100-connect-your-database.mdx]:
```suggestion
If you see the following error: `Error in connector: SCRAM failure: Authentication failed.`, you can specify the source database for the authentication by [adding](https://github.com/prisma/prisma/discussions/9994#discussioncomment-1562283) `?authSource=admin` to the end of the connection string.

Comment (tanberry) [reply content/100-getting-started/02-setup-prisma/200-add-to-existing-project/120-mongodb/100-connect-your-database.mdx]: We do have exactly this issue documented in our [Upgrade from Prisma 1 docs](https://www.prisma.io/docs/guides/upgrade-guides/upgrade-from-prisma-1/upgrade-from-mongodb-beta#introspect-your-mongodb-database).... and it sounds from the discussion like it is still a problem users might run into. I agree this is a good place to add it.. eventually we might have a general MongoDB troubleshooting section.
Thanks @zub0r for this PR, please review my suggested changes.

Comment (zub0r) [reply content/100-getting-started/02-setup-prisma/200-add-to-existing-project/120-mongodb/100-connect-your-database.mdx]: Sounds good, thanks.

**Review by nikolasburk — APPROVED** [review]

File: content/100-getting-started/02-setup-prisma/200-add-to-existing-project/120-mongodb/100-connect-your-database.mdx [hunk 2]
```diff
@@ -40,6 +40,8 @@ Here's a short explanation of each component:
 - `PORT`: The port where your database server is running (typically `27017` for MongoDB)
 - `DATABASE`: The name of the database
 
+If you see the following error: `Error in connector: SCRAM failure: Authentication failed.`, you can specify the source database for the authentication by [adding](https://github.com/prisma/prisma/discussions/9994#discussioncomment-1562283) `?authSource=admin` to the end of the connection string.
```
Comment (nikolasburk) [inline content/100-getting-started/02-setup-prisma/200-add-to-existing-project/120-mongodb/100-connect-your-database.mdx]:
```suggestion
> If you see the following error: `Error in connector: SCRAM failure: Authentication failed.`, you can specify the source database for the authentication by [adding](https://github.com/prisma/prisma/discussions/9994#discussioncomment-1562283) `?authSource=admin` to the end of the connection string.
```

**Comment by nikolasburk** [comment]
Thanks for taking the time to help us improve our docs @zub0r (and apologies that it took so long to get this merged 😅 ).

**Pull request merged**